etc/kayobe/environments/stackhpc-baremetal/kolla/config/ironic/ironic.conf

```diff
@@ -0,0 +1,74 @@
+{% raw %}
```

Comment:
we need to move this out of the ironic folder, ironic.conf just lives under /config.